client/me/purchases/purchases-site/reconnect-notice.jsx

```diff
@@ -0,0 +1,48 @@
+/**
+ * External dependencies
+ */
+import { localize } from 'i18n-calypso';
+import React, { Component } from 'react';
+
+/**
+ * Internal dependencies
+ */
+import Notice from 'components/notice';
+import NoticeAction from 'components/notice/notice-action';
+import support from 'lib/url/support';
+
+class PurchaseReconnectNotice extends Component {
+	static propTypes = {
+		name: React.PropTypes.string,
+		domain: React.PropTypes.string,
+	}
+
+	render() {
+		const { translate, name, isJetpack } = this.props;
+		let text = translate( 'You are no longer a user on %(site)s and cannot manage this purchase.', {
+			args: {
+				site: name,
+			}
+		} );
+		if ( isJetpack ) {
+			text = translate( '%(site)s has been disconnected from WordPress.com.', {
```

Comment:
Hi! I've found a possible matching string that has already been translated **17** times:
`translate( '%s has been disconnected.' )` *ES Score: 7*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).